response.go

```diff
@@ -35,6 +80,16 @@ type SQLInfo struct {
 	InfoAutoincrementIds []uint64
 }
 
+// Header is a response header.
+type Header struct {
+	// RequestId is an id of a corresponding request.
+	RequestId uint32
+	// Code is a response code.
+	// Code == OkCode if there is no error.
+	// Otherwise, it contains an error code.
+	Code int
+}
```

Comment:
Let's move the structure into `header.go` file. It would be easy to search in the future.
```suggestion
// Header is a response header.
type Header struct {
	// RequestId is an id of a corresponding request.
	RequestId uint32
	// Code is a response code.
	// Code == OkCode if there is no error.
	// Otherwise, it contains an error code.
	Code int
}
```

Reply:
Updated.